plugins/modules/aci_cloud_subnet.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, nkatarmal-crest <nirav.katarmal@crestdatasys.com>
+# Copyright: (c) 2020, Cindy Zhao <cizhao@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_cloud_subnet
+short_description: Manage Cloud Subnet (cloud:Subnet)
+description:
+- Manage Cloud Subnet on Cisco Cloud ACI.
+notes:
+- More information about the internal APIC class B(cloud:Subnet) from
+  L(the APIC Management Information Model reference,https://developer.cisco.com/docs/apic-mim-ref/).
+author:
+- Nirav (@nirav)
+- Cindy Zhao (@cizhao)
+options:
+  name:
+    description:
+    - The name of the Cloud Subnet.
+    type: str
+    aliases: [subnet]
+  description:
+    description:
+    - Description of the Cloud Subnet.
+    type: str
+  address:
+    description:
+    - Ip address of the Cloud Subnet.
+    type: str
+  name_alias:
+    description:
+    - An alias for the name of the current object. This relates to the nameAlias field in ACI and is used to rename object without changing the DN.
+    type: str
+  scope:
+    description:
+    - capability domain
+    choices: [ private, public, shared ]
+    type: str
+  usage:
+    description:
+    - usage of the port
+    choices: [ infra-router, user ]
+    type: str
```

Comment:
Is this for the VGW/VNG? I think we should expose it like the UI and not like the API.